org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -528,4 +531,130 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 		#{"**/*.xtend","**/*.mwe2"}
 	}
 	
+	/**
+	 * @since 2.11
+	 */
+	def private getWfLaunchConfigFile() {
```

Comment:
I'm fine with the suggestion. The name was chosen since it was the same method name before. But since it is private it is fair enough to make it more readable. I'll change that.